Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1511,7 +1657,30 @@ - (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range r
     return returnValue;
 }
 
+- (void)textViewDidChangeSelectionV2:(UITextView *)textView {
+    NSRange range = textView.selectedRange;
+    if (range.length > 0) {
+        return;
+    }
+    // Find a mentions query from the last control char if there is one
+    NSString *query = [self mentionsQuery:textView.text location:range.location];
+    if (query) {
+        // query it
+        [self beginMentionsCreationWithString:query
+                                   atLocation:range.location
+                        usingControlCharacter:YES
+                             controlCharacter:[@"@" characterAtIndex:0]];
```

Comment:
control char should be passed in by the consumer of the library rather than hard coding it right?

Reply:
yes upcoming rb